.gitignore

```diff
@@ -16,4 +16,6 @@ mps-prj/solutions/my.build/build.xml
 test/generate-build-solution/mps-prj/solutions/my.build/build.xml
 .DS_Store
 workspace.xml
+**/source_gen**
+**/classes_gen/**
```

Comment:
This seems inconsistent with the line above.
Maybe just ignore `**/*_gen*/`?

Reply:
Because there could also be `test_gen`

Reply:
I'd suggest `**/*_gen/` then so that `my_fancy_file_with_gen_in_name.txt` is not included

Reply:
a file won't be included, only a directory. There's also `source_gen.caches/`

Reply:
good to know